functionalDB/functionalDB.markdown

```diff
@@ -585,50 +585,50 @@ At this point we can see that with few minor tweaks, we can also provide a way t
 what-if → _transact →   _what-if → transact-on-db
 
 * The user calls `what-if` with the database value and the operations to perform. It then relays these inputs to `_transact`, adding to them a function that mimics `swap!`'s APIs, without its effect (callled `_what-if`).  
-````clojure
+```clojure
 (defmacro what-if [db & ops]  `(_transact ~db _what-if  ~@ops))
-````
+```
 * `_transact` prepares the call to `_what-if`. It does so by creating a list that begins with `_what-if`, followed by the database, then the `transact-on-db` symbol and the batch of operations.
 * `_what-if` invokes `transact-on-db`, just like `swap!` does in the transaction scenario, but does not inflict any change on the system.
 
-````clojure
+```clojure
 (defn- _what-if [db f txs]  (f db txs))
-````
+```
  
 Note that we are not using functions, but macros. The reason for using macros here is that arguments to macros do not get evaluated as the call happens; this allows us to offer a cleaner API design where the user provides the operations structured in the same way that any function call is structured in Clojure. 
 
 The above process can be seen in the following examples.
 
 For Transaction, the user call: 
-````clojure
+```clojure
 (transact db-conn  (add-entity e1) (update-entity e2 atr2 val2 :db/add))  
-````
+```
 changes into: 
-````clojure
+```clojure
 (_transact db-conn swap! (add-entity e1) (update-entity e2 atr2 val2 :db/add))
-````
+```
 which becomes: 
-````clojure
+```clojure
 (swap! db-conn transact-on-db [[add-entity e1][update-entity e2 atr2 val2 :db/add]])
-````
+```
 
 The what-if user call:

```

Comment:
To keep it structured the same way as transaction, how about changing this sentence to be "For What-if, the user call"